tests/ra_aid/test_llm.py

```
@@ -63,7 +63,8 @@ def test_initialize_expert_openai_custom(clean_env, mock_openai, monkeypatch):
     _llm = initialize_expert_llm("openai", "gpt-4-preview")
 
     mock_openai.assert_called_once_with(
-        api_key="test-key", model="gpt-4-preview", temperature=0
+        api_key="test-key", model="gpt-4-preview", temperature=0, reasoning_effort="high",
+        timeout=180, max_retries=5
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)